Gem/Code/Source/StreamingImageExampleComponent.cpp

```diff
@@ -457,8 +457,15 @@ namespace AtomSampleViewer
             if (m_3dImages.size() == 0)
             {
                 Create3dimages();
+                if (m_3dImages.size() == 0)
```

Comment:
we already have if (m_3dImages.size() == 0) above so no need for it here.

Reply:
The image may be created failed. And that’s the second condition for.